directory/pyproject.toml

```diff
@@ -39,7 +39,7 @@ dynamic = [
 
 [project.optional-dependencies]
 deps = [
-    "scandir==1.10.0",
+    "scandir==1.10.0; python_version < '3.5'",
```

Comment:
stupid question here maybe but: why 3.5 if they claim they support 3.7?

Reply:
Ah, I should probably comment on this: in Python 3.5 `scandir` function was added to the standard lib!
great q!